### ISSUE TITLE
pkl-models are deprecated in libvmaf 2.0.0+, cannot use json-models!

### ISSUE THREAD

**Issue by e-d-n-a** [fifonik/FFMetrics#10]
### Loading a pkl-model fails with latest ffmpeg-build
Taken from FFMetrics.log:
```
libvmaf WARNING could not read model from path: "D:/[...]/FFMetrics/vmaf-models/vmaf_v0.6.1.pkl"
libvmaf WARNING pkl model files have been deprecated, use json
libvmaf ERROR problem loading model file: D:/[...]/FFMetrics/vmaf-models/vmaf_v0.6.1.pkl
[Parsed_libvmaf_3 @ 000000000b1511c0] libvmaf encountered an error, check log for details
```

I tested 2 recent ffmpeg-build from [gyan.dev](https://www.gyan.dev/ffmpeg/builds/packages/):
```
ffmpeg-4.3.1-2021-01-01-full_build.7z
ffmpeg-4.3.1-2020-11-19-full_build.7z
```

The older one (2020-11-19) works with pkl-models!

The latest one (2021-01-01) now probably uses libvmaf v2, as it got [released 2020-12-04](https://github.com/Netflix/vmaf/releases/tag/v2.0.0) and doesn't support pkl-models anymore!
You can [see in the code](https://github.com/Netflix/vmaf/blob/9db0c56ce6d4019456a84828f9ed8d8d9da0ff09/libvmaf/src/model.c#L213), that it only tries to load json-models now and it at least [throws warnings since v2.1.0](https://github.com/Netflix/vmaf/blob/2e1b24d8344c8e8dbb770699ac5beae8b11463d4/libvmaf/src/model.c#L275)!

### Unable to use json-models

After putting recent json-models into the models-folder, they cannot be selected in the model-dopdown-menu of "VMAF options" in the GUI.
I also tried to load a json-model by changing my "FFMetrics.conf", but it doesn't use the specified default model name as can be seen from FFMetrics.log.

**Comment by fifonik** [comment]
The program is not expecting .json files right now. 
I will fix this the next version.
Thanks.

**Comment by e-d-n-a** [comment]
OK. Please also support choosing json-models via "FFMetrics.conf", which currently doesn't work. See #9.
This would allow to modify the filter-graphs and for me to use my "FFMetrics.conf" for automatic upscaling in future releases!
TY :)

**Comment by fifonik** [comment]
By design, you should be able to replace "{{model_path}}" in the conf with direct path to the model.
Unfortunately, GUI will know nothing about this and so the model dropdown will not be disabled in this case (it is possible to change this, but it is very low priority as the main idea is to use GUI to select model).

If I remember correctly, the conf was not processed correctly. I've found this issue recently. Should be fixed in the next version (I will double check it).

**Comment by fifonik** [comment]
Hm. It is more complicated than I thought. I have no idea how to detect what libvmaf version was used for building ffmpeg.

**Comment by fifonik** [comment]
json models can be used in version 0.9.0

**Comment by fifonik** [comment]
I've changed my mind and decided to add supported model format auto-detection (json or pkl) on startup. Already implemented and will be available in the next version.

**Comment by fifonik** [comment]
VMAF model type auto-detection implemented in version 0.9.5b.
